.github/workflows/documentation.yml

```diff
@@ -0,0 +1,48 @@
+name: Deploy Documentation to GitHub Pages
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - main
```

Comment:
it would be good idea to run all the steps, except upload artifact one every time.

Reply:
Yeah, action should now always run build test (for a push in any branch), but will only do deployment steps if this is main branch at dovecot/imaptest.